recipe/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "tensorboard-data-server" %}
+{% set version = "0.6.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/74/69/5747a957f95e2e1d252ca41476ae40ce79d70d38151d2e494feb7722860c/tensorboard_data_server-0.6.1-py3-none-any.whl
+  sha256: 809fe9887682d35c1f7d1f54f0f40f98bb1f771b14265b453ca051e2ce58fca7
+
+build:
+  script: "python -m pip install tensorboard_data_server-{{ version }}*.whl --no-deps --ignore-installed -vv "
+  number: 1
+  noarch: python
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  imports:
+    - tensorboard_data_server
+
+about:
+  home: http://tensorflow.org/
+  license: Apache-2.0
+  license_family: Apache
+  #  license_file: LICENSE
```

Comment:
Is there no license file in the repo?

Reply:
We are installing from pip package, and it doesn't have LICENSE file.